src/pivot-table/components/DimensionCell.tsx

```diff
@@ -39,19 +43,15 @@ const DimensionCell = ({ model, cell, isLeftColumn = false, rowIndex = 0, colInd
       : () => model.collapseTop(PATH, rowIndex, colIndex, false);
   }
 
-  const DimCell = (
-    <View style={style}>
-      <Text
-        style={[sharedStyles.text, styles.text]}
-        numberOfLines={1}>
-          {cellContent}
-      </Text>
-    </View>
+  return (
+    <div style={{ ...style, ...containerStyle}}>
```

Comment:
Would it make any sense creating a real table now that we don't do mobile native anymore?

Reply:
Not while we are using the `react-window` component. There each cell get an absolute position. Which allows us to only render part of a larger dataset.